Core/Newsletter2Go/Views/backend/newsletter2go/view/cart.js

```diff
@@ -0,0 +1,129 @@
+//{namespace name=backend/newsletter2go/view/tracking}
+//{block name="backend/newsletter2go/view/tracking"}
+Ext.define('Shopware.apps.Newsletter2go.view.Cart', {
+    extend: 'Ext.form.FieldSet',
+    alias: 'widget.cart-nl2go',
+    collapsible: true,
+    collapsed: false,
+    hidden: false,
+    width: '100%',
+    margin: 5,
+    border: true,
+    autoScroll: false,
+    defaults: {
+        labelWidth: 160,
+        anchor: '100%'
+    },
+    snippets: {
+        title: '{s name=cart/title}Cart Tracking{/s}'
+    },
+    initComponent: function () {
+        var me = this,
+            data = me.record;
+        me.disabled = !data['testConnection'];
+        me.title = me.snippets.title;
+        me.items = me.createForm();
+        me.registerEvents();
+
+        me.callParent(arguments);
+    },
+    registerEvents: function () {
+        this.addEvents('cartTracking');
+        this.addEvents('savePreferences');
+    },
+    createForm: function () {
+        var me = this,
+            labelText,
+            buttonText,
+            data = me.record;
+        if (data['trackCarts'] === '1') {
+            labelText = ' Enabled';
+            buttonText = 'Disable Tracking';
+        } else {
+            labelText = ' Disabled';
+            buttonText = 'Enable Tracking';
+        }
+
+        let mailingCombobox = Ext.create('Ext.form.field.ComboBox', {
+            name: 'mailing',
```

Comment:
it looks like that item Id is missed.
UI did not work for me properly with out this line. Since the id is not set but used at Core/Newsletter2Go/Views/backend/newsletter2go/view/cart.js:122

or here: 
https://github.com/newsletter2go/newsletter2go-shopware5-plugin/pull/23/files#diff-5b50fa9d332150ff4f36200d0b9326a3R122

```
itemId: 'nl2goCartMailingsCombobox',
```